packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -0,0 +1,295 @@
+  padding: ${GEL_SPACING_THREE_QUARTERS};
+  vertical-align: middle;
+`;
+
+const StyledSummary = styled.p`
+  ${({ script }) => script && getLongPrimer(script)};
+  ${({ service }) => service && getSansRegular(service)}
+  color: ${C_SHADOW};
+  margin: 0; /* Reset */
+  padding: ${GEL_SPACING} 0 ${GEL_SPACING_DBL};
+`;
+
+const StyledHeading = styled.h3`
+  ${({ script }) => script && getPica(script)}
+  ${({ service }) => getSansBold(service)}
+  color: ${C_EBON};
+  margin: 0; /* Reset */
+`;
+
+const HeadingLink = styled.a`
+  position: static;
+  color: ${C_EBON};
+  text-decoration: none;
+
+  &:before {
+    bottom: 0;
+    content: '';
+    left: 0;
+    overflow: hidden;
+    position: absolute;
+    right: 0;
+    top: 0;
+    white-space: nowrap;
+    z-index: 1;
+  }
+
+  &:visited {
+    color: ${C_METAL};
+  }
+
+  :hover,
+  :focus {
+    text-decoration: underline;
+  }
+`;
+
+const StyledLiveIndicator = styled.span`
+  color: ${C_POSTBOX};
+  text-transform: uppercase;
+  text-decoration: none;
+  :hover {
+    text-decoration: none;
+  }
+`;
+
+const BulletinBodyWrapper = styled.div`
+  padding-top: ${GEL_SPACING};
+`;
+
+const BulletinImageWrapper = styled.div`
+  padding-bottom: ${GEL_SPACING};
+`;
+
+const StyledBulletinImage = styled.img`
+  display: block;
+  width: 100%;
+  margin: 0;
+  padding: 0;
+`;
+
+const BulletinBody = ({
+  title,
+  summary,
+  type,
+  liveText,
+  isLive,
+  script,
+  service,
+  link,
+  ctaText,
+  hiddenTitleText,
+  dir,
+}) => (
+  <BulletinBodyWrapper dir={dir}>
+    <StyledHeading script={script} service={service}>
+      <VisuallyHiddenText>{hiddenTitleText}</VisuallyHiddenText>
+      {isLive && <LiveIndicator text={liveText} />}
+      <HeadingLink href={link} dir={dir}>
+        {title}
+      </HeadingLink>
+    </StyledHeading>
+    <StyledSummary script={script} service={service}>
+      {summary}
+    </StyledSummary>
+    <PlayCTA
+      type={type}
+      isLive={isLive}
+      dir={dir}
+      service={service}
+      script={script}
+    >
+      <IconWrapper>{mediaIcons[type.toLowerCase()]}</IconWrapper>
+      {ctaText}
+    </PlayCTA>
+  </BulletinBodyWrapper>
+);
+
+BulletinBody.propTypes = {
+  title: string.isRequired,
+  summary: string.isRequired,
+  type: oneOf(['video', 'audio']).isRequired,
+  liveText: string,
+  script: shape(scriptPropType).isRequired,
+  service: string.isRequired,
+  link: string.isRequired,
+  ctaText: string.isRequired,
+  hiddenTitleText: string.isRequired,
+  dir: oneOf(['ltr', 'rtl']).isRequired,
+  isLive: bool.isRequired,
+};
+
+BulletinBody.defaultProps = {
+  liveText: null,
+};
+
+const LiveIndicator = ({ text }) => (
+  <StyledLiveIndicator>{text}</StyledLiveIndicator>
+);
+
+LiveIndicator.propTypes = {
+  text: string.isRequired,
+};
+
+export const RadioBulletin = ({
+  title,
+  summary,
+  liveText,
+  dir,
+  script,
+  service,
+  link,
+  ctaText,
+  hiddenTitleText,
+  isLive,
+}) => (
+  <BulletinWrapper dir={dir}>
```

Comment:
Do we need to pass the `dir` prop to the `BulletinWrapper` and `BulletinBodyWrapper`?

Reply:
Doesn't look like it. Removed.